### PR TITLE
Patch 1.2 suggestion

### PR DESCRIPTION
**Pull request by daniel-heatwave**
Patch `1.1` has some problems, so I patched some up :D

### CODE REVIEW

File: userconfig.cfg [hunk 1]
```diff
@@ -15,6 +15,7 @@ exec object.cfg
 exec quickuse.cfg
 exec dd.cfg
 exec duckroll.cfg
+exec usespam.cfg
```
Comment (barney-heatwave) [inline userconfig.cfg:18]: Why has this been added? This doesn't exist.

Comment (daniel-heatwave) [reply userconfig.cfg:18]: `object.cfg` doesn't work it says `+obbo unknown command` in console :|

**Comment by barney-heatwave** [comment]
I'll test this and see if it works.

**Comment by barney-heatwave** [comment]
This is great, thanks! I will add this to release `1.1.1`

**Pull request merged**